README.md

```diff
@@ -61,8 +61,15 @@ If you choose to do this make sure you update the `data-generator.yaml` file for
 6. Deploy Flink Kubernetes Operator 1.10.0 (the latest stable version):
    ```
    helm repo add flink-operator-repo https://downloads.apache.org/flink/flink-kubernetes-operator-1.10.0/
-   helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator --set podSecurityContext=null -n flink
+   helm install flink-kubernetes-operator flink-operator-repo/flink-kubernetes-operator \
+   --set podSecurityContext=null \
+   --set defaultConfiguration."log4j-operator\.properties"=monitorInterval\=30 \
+   --set defaultConfiguration."log4j-console\.properties"=monitorInterval\=30 \
+   -n flink
    ```
+   Note:<br>
+   (1) Set `podSecurityContext` to null so that we can run in OpenShift environment<br>
+   (2) Set `monitorInterval` to log4j properties file so that we can dynamically change log level for operator and job/task manager.
```

Comment:
The monitorInterval is also only for Openshift environment?

Reply:
No, it's the property for log4j, not related to the running env.